test/JDBC/input/sp_describe_first_result_set-vu-prepare.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE test_table1(a int, b int);
+GO
+
+CREATE PROCEDURE sp_describe_first_result_set_vu_prepare_procedure
```

Comment:
I am not sure if this testing is sufficient, specially can we have use case where some object depends on this procedure and we cannot drop it. We are doing rename to handle that but there is no test case to verify the same.